api/client/client_test.go

```diff
@@ -412,12 +419,15 @@ func TestListen(t *testing.T) {
 		if val.err != nil {
 			t.Fatalf("failed to receive first listen result: %v", val.err)
 		}
-		p := &Person{}
-		if err := json.Unmarshal(val.data, p); err != nil {
-			t.Fatalf("failed to unmarshal listen result: %v", err)
-		}
-		if p.Age != 30 {
-			t.Fatalf("expected listen result age = 30 but got: %v", p.Age)
+		// p := &Person{}
+		// if err := json.Unmarshal(val.data, p); err != nil {
+		// 	t.Fatalf("failed to unmarshal listen result: %v", err)
+		// }
+		// if p.Age != 30 {
+		// 	t.Fatalf("expected listen result age = 30 but got: %v", p.Age)
+		// }
```

Comment:
Possibly need to add some checks in here on the contents of the actual data. This will depend on how we decide to provide the data. Either we send it along in the event or we do a request for the data here from the client.

Reply:
Ended up deciding to send the entity data along with the event so this part of the test is re-enabled.